api/server/router/local/local.go

```diff
@@ -0,0 +1,161 @@
+package local
```

Comment:
Cool, I was going to ask that question as well. I think that would be good. Thanks!